src/main/java/seedu/address/logic/commands/RedoCommand.java

```diff
@@ -28,7 +29,7 @@ public RedoCommand() {
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
-        logger.log(Level.INFO, "Going to execute redo command.");
+        logger.log(Level.INFO, LOGGER_MESSAGE);
```

Comment:
should it be more specific, like LOGGER_REDO_MESSAGE?

Reply:
resolved